bigbluebutton-html5/public/locales/en.json

```
@@ -1014,6 +1014,7 @@
     "app.toast.wakeLock.disclaimer": "{error}. You will be dropped from the call after your screen turns off.",
     "app.toast.setEmoji.raiseHand": "You have raised your hand",
     "app.toast.setEmoji.lowerHand": "Your hand has been lowered",
+    "app.toast.raisedHandNext.label": "You're next on the list with your hand raised.",
```

I suggest this change as "line" conveys an order, while "list" can imply items without sequence

```suggestion
    "app.toast.raisedHandNext.label": "You're next in line with your hand raised",
```